RecoBTag/ONNXRuntime/interface/tensor_configs.h

```diff
@@ -28,4 +30,85 @@ namespace deepvertex {
 
 }  // namespace deepvertex
 
+namespace parT {
+
+  enum InputFeatures {
+    kBegin=0,
+    kChargedCandidates=kBegin,
+    kNeutralCandidates=1,
+    kVertices=2,
+    kChargedCandidates4Vec=3,
+    kNeutralCandidates4Vec=4,
+    kVertices4Vec=5,
+    kEnd=6
+  };
+
+  constexpr unsigned n_cpf_accept = 25;
+  constexpr unsigned n_npf_accept = 25;
+  constexpr unsigned n_sv_accept = 5;
+
+  const std::map<InputFeatures, unsigned int> N_InputFeatures{
+    {kChargedCandidates, 16},
```

Comment:
these could actually be vectors rather than maps, since the enum is ordered and dense. that would speed up some of the operations in the producers.